### ISSUE TITLE
[Feature Request]: allow configure in raw source code in Code panel

### ISSUE THREAD

**Issue by maydayenenen** [storybookjs/storybook#21608]
### Is your feature request related to a problem? Please describe

I am currently working on a project using storybook which add Code panel and Controls panel. This is what I have in mdx file now:

```
export const Template = ({dark, width, height, theme}) => {
  return html`
    <my-button
      @click=${action('button clicked')}
      width="${width}"
      height="${height}"
      theme="${theme}"
      ?dark=${dark}>
    </my-button>
  `;
};
```

This is Code panel when the user set `dark` to true in Control panel: 

`<my-button  width="200" height="40" theme="dark"  dark="">
</my-button>`

**dark** is a boolean user can set in Control panel. 

We're supporting displaying raw source code in Code panel for each component. The value of the attribute for the html element will change based on different configuration of the component. But we found some
limitations of overriding the source code in Code panel.

What we have now:
- Dark mode on : `<my-button dark=""></my-button>`
- Dark mode off : `<my-button ></my-button>`

We want to update all of our components to the following:

- Dark mode on : `<my-button dark></my-button>`
- Dark mode off : `<my-button ></my-button>`

Setting `dark` without a value works as expected, but removing `=""` is not achievable. We can define what type of inputs to display, but we don't have control over how this particular playground tab displays them, and there doesn't seem to be a type option for **no-value** attributes. No matter what we passed as a value to the attribute, it will be parsed to `=""`. 

### Describe the solution you'd like

We need to update all of our components to the following:

- Dark mode on : `<my-button dark></my-button>`
- Dark mode off : `<my-button ></my-button>`

### Are you able to assist to bring the feature to reality?

no

**Comment by shilman** [comment]
Is this a web-components project? If so this could also be a bug in the web components source snippet generation, since `<my-button dark></my-button>` should be the behavior out of the box.

**Comment by maydayenenen** [comment]
@shilman  yes, it is a web-component project. Could you give me more information?

**Comment by liu351** [comment]
I see this as well with storybook web-components. Wondering what we can do for any undefined args so that an attribute wont be in the code snippet if not provided to avoid <custom-element my-attribute="">
